packages/react-components/src/molecules/SearchField.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ComponentProps } from 'react';
+import css from '@emotion/css';
+
+import { TextField } from '../atoms';
+import { perRem } from '../pixels';
+import { searchIcon } from '../icons';
+
+const containerStyles = css({
+  paddingBottom: `${18 / perRem}em`,
+});
+
+type SearchFieldProps = {} & Exclude<ComponentProps<typeof TextField>, 'id'>;
```

Comment:
This should probably whitelist (Pick, not Exclude) only what we need for our use case, most of the `TextField` props don't make sense on the search